ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -345,6 +345,11 @@ def minimum(self, y, name=None):
         y_ivy = _to_ivy_array(y)
         return ivy.minimum(self._ivy_array, y_ivy)
 
+    @with_supported_dtypes({"2.5.0 and below": ("int32", "int64")}, "paddle")
+    def floor_divide(self, y, name=None):
+        y_ivy = _to_ivy_array(y)

```

Comment:
Apologies for the confusion. You can use something like: `y_ivy = y._ivy_array if isinstance(y, Tensor) else _to_ivy_array(y)`. I think that should resolve the issue

Reply:
Thank you @hmahmood24, I have done a rebase to upstream master and the requested update please review and get back.